cmd/cmd.go

```diff
@@ -520,7 +515,6 @@ func generateInteractive(cmd *cobra.Command, model string) error {
 
 	completer := readline.NewPrefixCompleter(
 		readline.PcItem("/help"),
-		readline.PcItem("/list"),
```

Comment:
Don't remove this part or the command. I'm in the middle of updating this.

Reply:
I removed the commit that removes this REPL command